transportable-udfs-test/transportable-udfs-test-spark_2.12/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'scala'
+
+sourceSets {
+  main {
+    scala {
+      srcDirs = project(':transportable-udfs-test:transportable-udfs-test-spark_2.11').sourceSets.main.scala.srcDirs
```

Comment:
spark_2.12 test with spark_2.11 srcDirs setting?

Reply:
Yup, we don't want to copy the code again if we can just compile it for scala_2.12. 

srcDirs achieves that.